protocol/oracle-manager/test/integration/modules/NodeModule.test.ts

```diff
@@ -60,4 +52,66 @@ describe('NodeModule', function () {
+      );
+    });
+  });
+
+  describe('processManyWithRuntime()', () => {
+    it('when everything is success, returns an array with all the good stuff', async () => {
+      const priceDatas = await NodeModule.processManyWithRuntime([nodeId2(), nodeId1()], [], []);
+
+      assertBn.equal(priceDatas[0].price, ethers.utils.parseEther('0.9'));
+      assertBn.equal(priceDatas[1].price, ethers.utils.parseEther('1.0'));
+    });
+
+    it('when fail, returns Errors() event', async () => {
+      await assertRevert(
+        NodeModule.processManyWithRuntime(
+          [failingNodeId(), nodeId2(), nodeId1(), failingNodeId()],
+          [],
+          []
+        ),
+        'Errors("0xac47be2100000000000000000000000000000000000000000000000000000000000004d2","0xac47be2100000000000000000000000000000000000000000000000000000000000004d2")',
+        NodeModule
+      );
+    });
+
+    it('when a nested nested node fails, returns correct Errors() event', async () => {
+      const NodeParameters = abi.encode(['uint'], [NodeOperations.MEDIAN]);
+      await NodeModule.registerNode(NodeTypes.REDUCER, NodeParameters, [nodeId1(), nodeId2()]);
+      const reducer1Id = await NodeModule.getNodeId(NodeTypes.REDUCER, NodeParameters, [
+        nodeId1(),
+        nodeId2(),
+      ]);
+      await NodeModule.registerNode(NodeTypes.REDUCER, NodeParameters, [reducer1Id, nodeId2()]);
+      const reducer2Id = await NodeModule.getNodeId(NodeTypes.REDUCER, NodeParameters, [
+        reducer1Id,
+        nodeId2(),
+      ]);
+
+      // cause an error after the fact on one of the internal nodes
+      await aggregator().mockSetFails(82828);
+
+      // should get the errors from the very deep reducer node, wrapped in two `Errors`. It will be parsed recursively outside
+      await assertRevert(
+        NodeModule.processManyWithRuntime([nodeId2(), nodeId1(), reducer2Id], [], []),
+        'Errors("0xac47be2100000000000000000000000000000000000000000000000000000000000004d2,0x0b42fd17000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000018000000000000000000000000000000000000000000000000000000000000001040b42fd1700000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000024ac47be2100000000000000000000000000000000000000000000000000000000000004d2000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")',
```

Comment:
can you please "encode" readable values in here to match against instead of hardcoding full response as is